domains/redchilly.is-cool.dev.json

```diff
@@ -0,0 +1,16 @@
+{
+  "description": "Self hosted website, and an app. ",
+
+  "domain": "is-cool.dev",
+  "subdomain": "redchilly",
+
+  "owner": {
+    "email": "opendomains@uk.is-a.dev"
+  },
+
+  "record": {
+    "NS": ["david.ns.cloudflare.com", "zita.ns.cloudflare.com"]
+  },
```

Comment:
Denied due to NS records. Please read pull request guidelines and truthfully certify before making PRs.
<img width="957" height="409" alt="image" src="https://github.com/user-attachments/assets/a7f72573-dc5d-4386-b1e0-04baed99f527" />